lib/src/client.dart

```diff
@@ -3880,6 +3888,11 @@ class SyncStatusUpdate {
 }
 
 enum SyncStatus {
+  notLoggedIn,
+  initializing,
+  initializingEncryption,
+  initializingDatabaseMigration,
+  initializingFinalize,
```

Comment:
This could be more descriptive, as a comment or maybe even a better naming scheme.